internal/action/sync.go

```diff
@@ -27,6 +28,7 @@ var (
 )
 
 func init() {
+	// TODO(gitconfig) use proper config setting
```

Comment:
I'm not sure to understand this todo. Wouldn't it be best to open an issue for it?